frontend/src/js/components/auditlogs/auditlogslist.js

```diff
@@ -43,6 +43,12 @@ const UserChange = ({ item: { change = '-' } }) => {
     </div>
   );
 };
+const TenantChange = props => {
+  const {
+    item: { change = '-' }
+  } = props;
+  return <div>{change}</div>;
+};
```

Comment:
just as this is so close to the user change now (where the diff splitting seems to be no longer needed) - could they be unified to something like a `ChangeFallback` component?